### ISSUE TITLE
Sbemu conflicting with usbuhcil usb dos drivers

### ISSUE THREAD

**Issue by TheUndertakerPoland** [crazii/SBEMU#85]
There is a game called blood.
So I have sbemu configured on wyse vx0 (via ac97)
Sound working.
I have ps2 keyboard and usb mouse.
Mouse running in Norton commander. I'm starting blood game showing mouse detected, but in the middle of the game mouse not working at all, sound works (tested with jemmex and without)
When I quit I have nice return to Norton commander but mouse not working until I restart usb drivers.

**Comment by crazii** [comment]
Thanks for reporting, currently I'm in the middle of improving compatibility with other devices (shared IRQ especially). I'll let you know when it is done.

Also, you can test if there's any chance that without sbemu and the mouse is not working, although I doubt there's little chance of it.

**Comment by TheUndertakerPoland** [comment]
Yes, without sound mouse working in the game. I noticed some progress I had version 4 days old and blood was crashing. Today installed latest version and looks like blood was running just mouse not mean together

**Comment by Ethaniel-404** [comment]
> There is a game called blood. So I have sbemu configured on wyse vx0 (via ac97) Sound working. I have ps2 keyboard and usb mouse. Mouse running in Norton commander. I'm starting blood game showing mouse detected, but in the middle of the game mouse not working at all, sound works (tested with jemmex and without) When I quit I have nice return to Norton commander but mouse not working until I restart usb drivers.

«usbuhcil» is the lite version. Try the standard «usbuhci» (it's in the same folder if you're using Bret Johnson's drivers), then usbmouse.com, and finally force PS2 mode on ctmouse with the /p modifier. I've had issues with «usbuhcil» as well.

**Comment by TheUndertakerPoland** [comment]
Usbuhci without l crashing in dos after run just power on off

**Comment by TheUndertakerPoland** [comment]
Also I have tried change irq and I must use irq 11 for usbuhcil and using sbemu with /i5 option even with that conflict exist I'm not very technical in old irq mapping but I can help with testing

**Comment by TheUndertakerPoland** [comment]
I don't have ps2 mouse ATM in hand :(

Wysłano z programu Outlook dla systemu Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Ethaniel-404** [comment]
Well, in that case, I believe @crazii has a project called USBDDOS. It is functionally similar to Bret Johnson's drivers, maybe you'll have better luck with it.

But, if you have PS/2 ports, _use them_. I'd drop all USB drivers in a heartbeat, but my netbook doesn't have any legacy ports.

**Comment by TheUndertakerPoland** [comment]
I'll check. I know I can connect ps2 mouse just ordered on amazon but all this play and digging in oldies is a lot of fun :) and I can help btw :)

**Comment by TheUndertakerPoland** [comment]
No joy with them. Usbddos seems to works but ctmouse not recognise any mouse, usbddosp crashing

**Comment by TheUndertakerPoland** [comment]
Wyse supporting legacy mode just ctmouse can't find any mouse straight so just fighting with usbdos

**Comment by Ethaniel-404** [comment]
Which version of ctmouse are you using? Remember, there's three branches for PS/2 handling. In other words, try the three of them (1.9.1, 2.0 alpha 4, and 2.1 beta 4).

**Comment by TheUndertakerPoland** [comment]
I have tested 1.9 2.0 and 2.1 and just 2.1 works for me. Works mean in combination of usbuhcil /irq11 and usbmouse /m:1 plus ctmouse all working fine.  I have legacy mode under bios and seems this works fine with pendrive (I was able to install full freedos from autoboot pendrive) I can enable/disable ehci mode. To the author there is a game what I noticed is hard to run with sound. It is Czech game called ramon's spell, I was able to run this under dosbox with some sb16 tricks and commands but seems like soundblaster option is greyed out with sbemu (for now ) :)

**Comment by crazii** [comment]
> No joy with them. Usbddos seems to works but ctmouse not recognise any mouse, usbddosp crashing

currently usbddos is working in  progress, and the release is behind the master branch code, you may try this one in the link
https://github.com/crazii/SBEMU/issues/13#issuecomment-1935415791
but it might not work either.
